src/lib/ui/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,64 @@
+import { type MouseEventHandler, useEffect, useState } from 'react'
+
+import { InfoIcon } from 'lib/icons/Info.js'
+import { InfoDarkIcon } from 'lib/icons/InfoDark.js'
+
+interface TooltipProps {
+  children: React.ReactNode
+}
+
+export function Tooltip({ children }: TooltipProps): JSX.Element {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const handleClose = (): void => {
+    setIsOpen(false)
+  }
+
+  const handleEscape = (ev: KeyboardEvent): void => {
+    if (ev.key === 'Escape') {
+      handleClose()
+    }
+  }
```

Comment:
Since these functions are a dep of `useEffect` you may need to wrap with `useCallback`.